app/services/record_declarations/actions/make_payable_between.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module RecordDeclarations
+  module Actions
+    class MakePayableBetween
+      include WithMilestone
+
+      def call
+        ParticipantDeclaration.eligible.where("declaration_date <= ?", milestone).each(&:make_payable)
+      end
+    end
+  end
+end
```

Comment:
Does this stuff belong in this PR?

Reply:
Not really. Cherry picking across 3 branches. I've taken this one out before. :)